target/linux/ramips/dts/mt7621_zyxel_keenetic-giga-iii.dts

```diff
@@ -0,0 +1,301 @@
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zyxel,keenetic-giga-iii", "mediatek,mt7621-soc";
+	model = "ZyXEL Keenetic Giga III";
+
+	aliases {
+		label-mac-device = &gmac0;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+	};
```

Comment:
This block can be dropped, it's in the DTSI.

Reply:
fixed